### PR TITLE
Revert fastai version change

### PR DESCRIPTION
**Pull request by twolodzko**
* For a reason unknown to me, the code with new version of fastai fails on Colab with the same exception as the code with old version of fastai was failing when running script locally. I'm reverting the version used in Colab.
* I also added import for the File that was missing in notebook.

Notice: the notebook fails.

### CODE REVIEW

**Comment by kamil-kaczmarek** [comment]
@twolodzko 

Can you fix it in a way that it will work in both script and colab?

**Comment by Blaizzy** [comment]
@twolodzko you are getting an error on Colab because Colab by default installs PIL version 7.x.x while fastai v2.7.2+ requires PIL v9.1.1 as specified here:

https://github.com/fastai/fastai/releases/tag/2.7.2

I think it would be better to pin the PIL version instead of reverting fastai version.

`!pip install ... pillow==9.1.1`

File: integrations-and-supported-tools/fastai/notebooks/Neptune_fastai.ipynb [hunk 1]
```diff
@@ -19,7 +19,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install fastai==2.7.2 neptune-client[fastai]"
+    "! pip install fastai==2.3.1 neptune-client[fastai]"
```
Comment (kamil-kaczmarek) [inline integrations-and-supported-tools/fastai/notebooks/Neptune_fastai.ipynb:22]:
```suggestion
    "! pip install fastai==2.6.3 neptune-client[fastai]"
```

**Review by kamil-kaczmarek — CHANGES_REQUESTED** [review]

**Comment by kamil-kaczmarek** [comment]
fastai tests passed, now problem is with LGBM tests -> to be fixed in separate PR.

**Review by kamil-kaczmarek — APPROVED** [review]

**Review by kamil-kaczmarek — APPROVED** [review]

**Comment by kamil-kaczmarek** [comment]
Colab shows some error in installation cell, yet notebook was computer successfully till the end.

**Pull request merged**